app/clients/email/aws_ses.py

```diff
@@ -106,14 +106,11 @@ def send_email(
             # http://docs.aws.amazon.com/ses/latest/DeveloperGuide/api-error-codes.html
```

Comment:
This URL doesn’t exist any more – equivalent seems to be [this one](https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_SendEmail.html#API_SendEmail_Errors) (which does document both `TooManyRequestsException` and (via the link to [common errors](https://docs.aws.amazon.com/ses/latest/APIReference-V2/CommonErrors.html)) `ThrottlingException`)
```suggestion
            # https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_SendEmail.html#API_SendEmail_Errors
```